File: python_library/loan.py
from datetime import datetime
from typing import Optional


class Loan:
    def __init__(self, loan_id: str, member_id: str, book_id: str, borrow_date: datetime):
        self.id = loan_id
        self.member_id = member_id
        self.book_id = book_id
        self.borrow_date = borrow_date
    
    def get_id(self) -> str:
        return self.id
    
    def get_member_id(self) -> str:
        return self.member_id
    
    def get_book_id(self) -> str:
        return self.book_id
    
    def get_borrow_date(self) -> datetime:
        return self.borrow_date

File: Loan.java
package com.library.management;

import java.util.Date;

public class Loan {
    private String id;
    private String memberId;
    private String bookId;
    private Date borrowDate;
    
    public Loan(String id, String memberId, String bookId, Date borrowDate) {
        this.id = id;
        this.memberId = memberId;
        this.bookId = bookId;
        this.borrowDate = borrowDate;
    }
    
    // Getters
    public String getId() { return id; }
    public String getMemberId() { return memberId; }
    public String getBookId() { return bookId; }
    public Date getBorrowDate() { return borrowDate; }
}
